azure/iac/disks.rego

```diff
@@ -6,9 +6,9 @@ package rule
 # PR-AZR-ARM-DSK-001
 #
 
-default disk_encrypt = null
+default disk_encryption = null
```

Comment:
@shahinshirvani if you are renaming the rule, make sure to rename variable parameter as well with the same name. example at line 19.

Reply:
fixed